src/python/pants/testutil/engine/util.py

```diff
@@ -159,8 +159,8 @@ def test_foo(self):
 
 
 def remove_locations_from_traceback(trace):
-  location_pattern = re.compile('"/.*", line \d+')
-  address_pattern = re.compile('0x[0-9a-f]+')
+  location_pattern = re.compile(r'"/.*", line \d+')
```

Comment:
what does this regex fix?

Reply:
DeprecationWarning that I found while testing if this fixes #6282. The original and new versions are identical, but the original will stop working in Python 3.9 I believe.